CHANGELOG.md

```diff
@@ -9,6 +9,34 @@ adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 ## [Unreleased]
 
 ---------------------------------------------------------
+
+## Added
+- Enable multi-column representations
+- Add `linear_split_with_rotation` splitting strategy
+- Added a Bayesian ordinal regression model.
```

Comment:
This probably refers to the ordinal_classifier in pystan, right? (I know the terms are a bit used interchangeably in this case - regressor/classifier), but maybe good to explicitly say the name of the added model here just to avoid confusion...

Reply:
Ah yeah! updated!